examples/zookeeper/Dockerfile

```diff
@@ -8,19 +8,21 @@ ENV ZOOKEEPER_VERSION 3.4.6
 
 EXPOSE 2181 2888 3888
 
-RUN  curl http://apache.mirrors.pair.com/zookeeper/zookeeper-${ZOOKEEPER_VERSION}/zookeeper-${ZOOKEEPER_VERSION}.tar.gz | tar -xzf - -C /opt \
+RUN curl http://apache.mirrors.pair.com/zookeeper/zookeeper-${ZOOKEEPER_VERSION}/zookeeper-${ZOOKEEPER_VERSION}.tar.gz | tar -xzf - -C /opt \
+    && yum update -y \
     && yum install -y gettext && yum clean all \
     && mv /opt/zookeeper-${ZOOKEEPER_VERSION} /opt/zookeeper \
     && cp /opt/zookeeper/conf/zoo_sample.cfg /opt/zookeeper/conf/zoo.cfg \
-    && mkdir -p /opt/zookeeper/{data,log}
+    && mv /opt/zookeeper/conf{,.template} \
+    && mkdir -p /opt/zookeeper/{conf,data,log}
 
 WORKDIR /opt/zookeeper
-VOLUME ["/opt/zookeeper/conf", "/opt/zookeeper/data", "/opt/zookeeper/log"]
 
 COPY config-and-run.sh ./bin/
-COPY zoo-template.cfg ./conf/
+COPY zoo-template.cfg ./conf.template/
+RUN chown -R jboss:jboss /opt/zookeeper
```

Comment:
since the image may not run as jboss, this should be chown -R jboss:0
https://github.com/openshift/jenkins/blob/master/1/Dockerfile#L50